README.md

```diff
@@ -65,9 +65,10 @@ vars:
 ### (Optional) Step 4: Additional configurations
 <details open><summary>Expand/Collapse configurations</summary>
 
-#### Note about currency gains
+#### Multi-currency Support Limitations
+Currently, our dbt models for Xero have limited support for multi-currency accounting, particularly for handling unrealized currency gains and losses and bank revaluations, as they require historical or current exchange rate data that is not present in the Xero connector. Xero relies on an external source ([XE.com](https://www.xe.com/)) or custom rates the customer adds for historical exchange rates, making it difficult to obtain that data.
 
-If you are using multi-currency accounting in Xero, you are likely to have unrealized currency gains as part of your profit and loss statement. These gains/losses do not exist within the actual journals in Xero. As a result, you will find that those lines are missing from the outputs of this package. All realised currency gains will be present and your balance sheet will still balance.
+These particular accounts are thus not available in the profit and loss statement. Your balance sheet will continue to balance correctly, reflecting all realized transactions. For accurate reporting of unrealized currency adjustments, an external source of historical exchange rate data would be required.
```

Comment:
This reads a bit strange. It sounds like we're saying these accounts aren't available, but your balance sheet will still balance correctly. Without these accounts however it won't balance properly and I don't want us to make the claim that it will in this scenario. Let's change this up a bit to make sure we mention that all realized current gains will be present, but the unrealized currency gains and losses and bank revaluations lines will not.

Reply:
Made some tweaks, let me know if this reads better.